{
  "commandAnnotations": [],
  "commands": [
    {
      "commandType": "home",
      "completedAt": "TIMESTAMP",
      "createdAt": "TIMESTAMP",
      "id": "UUID",
      "key": "50c7ae73a4e3f7129874f39dfb514803",
      "notes": [],
      "params": {},
      "result": {},
      "startedAt": "TIMESTAMP",
      "status": "succeeded"
    }
  ],
  "config": {
    "apiVersion": [
      2,
      16
    ],
    "protocolType": "python"
  },
  "createdAt": "TIMESTAMP",
  "errors": [
    {
      "createdAt": "TIMESTAMP",
      "detail": "InvalidTrashBinLocationError [line 15]: Invalid location for trash bin: C2.\nValid slots: Any slot in column 1 or 3.",
      "errorCode": "4000",
      "errorInfo": {},
      "errorType": "ExceptionInProtocolError",
      "id": "UUID",
      "isDefined": false,
      "wrappedErrors": [
        {
          "createdAt": "TIMESTAMP",
          "detail": "opentrons.protocol_api.validation.InvalidTrashBinLocationError: Invalid location for trash bin: C2.\nValid slots: Any slot in column 1 or 3.",
          "errorCode": "4000",
          "errorInfo": {
            "args": "('Invalid location for trash bin: C2.\\nValid slots: Any slot in column 1 or 3.',)",
            "class": "InvalidTrashBinLocationError",
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line N, in exec_run\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line N, in <module>\n\n  File \"Flex_X_v2_16_NO_PIPETTES_TrashBinInCol2.py\", line N, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line N, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line N, in load_trash_bin\n    addressable_area_name = validation.ensure_and_convert_trash_bin_location(\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/validation.py\", line N, in ensure_and_convert_trash_bin_location\n    raise InvalidTrashBinLocationError(\n"
          },
          "errorType": "PythonException",
          "id": "UUID",
          "isDefined": false,
          "wrappedErrors": []
        }
      ]
    }
  ],
  "files": [
    {
      "name": "Flex_X_v2_16_NO_PIPETTES_TrashBinInCol2.py",
      "role": "main"
    }
  ],
  "labware": [],
  "liquidClasses": [],
  "liquids": [],
  "metadata": {
    "author": "Derek Maggio <derek.maggio@opentrons.com>",
    "protocolName": "QA Protocol - Analysis Error - Trash Bin in Column 2"
  },
  "modules": [],
  "pipettes": [],
  "result": "not-ok",
  "robotType": "OT-3 Standard",
  "runTimeParameters": []
}
